src/scss/utilities/_wrapper.scss

```diff
@@ -4,8 +4,8 @@
 .wrapper {
   margin-left: auto;
   margin-right: auto;
-  width: 80rem; // For no clamp support
-  width: clamp(16rem, 95vw, 80rem);
+  width: 85rem; // For no clamp support
```

Comment:
Do we go wider wrapper again?

Reply:
Yeh we bumped it up a touch to allow for the better typesetting in the hero etc